socorro/unittest/external/es/test_crashstorage.py

```diff
@@ -207,29 +192,25 @@ def __init__(self, *args, **kwargs):
         )
         self.index_client = elasticsearch.client.IndicesClient(self.es_client)
 
-    def tearDown(self):
-        """Remove indices that may have been created by the test.
-        """
+    def teardown_method(self, method):
+        """Remove indices that may have been created by the test """
         try:
-            self.index_client.delete(
-                self.config.elasticsearch.elasticsearch_index
-            )
+            self.index_client.delete(self.config.elasticsearch.elasticsearch_index)
 
         except elasticsearch.exceptions.NotFoundError:
             # It's fine it's fine; 404 means the test didn't create any
             # indices, therefore they can't be deleted.
             pass
+        super(IntegrationTestESCrashStorage, self).teardown_method(method)
 
     def test_index_crash(self):
-        """Test indexing a crash document.
-        """
-
+        """Test indexing a crash document"""
         es_storage = ESCrashStorage(config=self.config)
 
         es_storage.save_raw_and_processed(
-            raw_crash=a_raw_crash,
+            raw_crash=deepcopy(a_raw_crash),
```

Comment:
These calls all need to deepcopy the raw crash and processed crash. The Elasticsearch crashstorage modifies the processed crash.